debian_server/privacyidea-apache2.postinst

```diff
@@ -114,6 +114,9 @@ create_database() {
 }
 
 enable_apache() {
+    if [ ! -h /etc/apache2/mods-enabled/rewrite.load ]; then
```

Comment:
We don't need the test here, `a2enmod` does nothing if the module is already enabled. Also the return value of `a2enmod` is still `0` in this case, so this should not have been the cause of the broken installation.